apps/api/src/modules/summary/summary.service.ts

```diff
@@ -31,9 +32,24 @@ export class SummaryService extends CoreService<Summary> {
     if (inputFile) {
       const { createReadStream, filename } = await inputFile;
 
-      const uniqueIdentifier = uuidv4();
+      const uniqueIdentifier = uuidv4().replace(/-/g, '').substring(0, 10);
       const modifiedFilename = `${uniqueIdentifier}_${filename}`;
-      const fileLocation = join(process.cwd(), `./src/upload/${modifiedFilename}`);
+      const uploadPath = process.env.Upload_Path
```

Comment:
Fix this, and we use configService not process.env.
defaultPath = join(process.cwd(), 'src', 'uploads');
Don't keep it inside src folder it should be in root along with logs process.cwd() + /uploads like I mentioned.

Reply:
Updated to use configservice